### ISSUE TITLE
[UI] Cluster Templates - UI still shows cluster template revision as disabled even if it fails to disable

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#21699]
Minor issue, but should address in v2.3.

Version: master-head (7/22/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Create a cluster template, you only need one revision.
- Leverage the template to create a cluster.
- Either while the cluster is still provisioning or fully ready, attempt to disable the (in-use) cluster template revision in the UI.

**Result:**
You will get an expected error message, but the issue is that the UI still shows the cluster template revision as Disabled, when it in fact is not disabled since this action failed.
Manually refresh the UI and then the template shows as Active again.

**Comment by soumyalj** [comment]
This issue needs to be fixed. Tested with master build.
Create a cluster template/revision T1/R1 and provide read-only access to user1 . 
Login as user1 and disable the template. User1 should not be allowed to disable the template, but the UI shows it as disabled although there is an error message. A browser refresh is required for the template to show active again.

**Comment by davidnuzik** [comment]
Version: master-head (8/2/19)

I follow the repro steps:
- Create a cluster template, you only need one revision.
- Leverage the template to create a cluster.
- Either while the cluster is still provisioning or fully ready, attempt to disable the (in-use) cluster template revision in the UI.

And I also tried what Soumya mentioned above https://github.com/rancher/rancher/issues/21699#issuecomment-515806944

In both cases the UI no longer shows the CTR as disabled even when it fails to disable. The growl error still pops up.

Pictured below, I attempted to disable the first "shared" template CTR. I got the expected growl message and now the UI still shows it as "Active" (not requiring a refresh anymore). This was Soumya scenario but both cases have the same result. This is fixed.
![image](https://user-images.githubusercontent.com/45179589/62391646-3e450a80-b51a-11e9-8d79-008f8bebb0eb.png)
